src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFeaturedVideoSound.java

```diff
@@ -53,7 +53,7 @@ private void verifyFeaturedVideoWithSound(String pageUrl) {
   }
 
   private void verifyFeaturedVideoWithoutSound(String pageUrl) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "verifyFeaturedVideoWithoutSound" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)